demo/config.js

```diff
@@ -0,0 +1,15 @@
+//Config file
+//TODO: Logic should be separated from configuration
+
+var config_data = {
+  'GENERAL_CONFIG': {
+    'END_POINT_BASE_URL': 'http://localhost:8079/blazegraph/'
+  }
+};
+
+var config_module = angular.module('demonstrator.config', []);
+
+angular.forEach(config_data,function(key,value) {
```

Comment:
I'm thinking this might be better in a [`module.run()`](https://docs.angularjs.org/api/ng/type/angular.Module#run) method (or maybe even [`module.config()`](https://docs.angularjs.org/api/ng/type/angular.Module#config), but I'm not sure if that works.